src/Tizen.NUI/src/public/Common/Color.cs

```diff
@@ -953,31 +953,90 @@ public Color(float[] array) : this(Interop.Vector4.NewVector4(ValueCheck(array))
         }
 
         /// <summary>
-        /// The conversion constructor from an hexcode of four floats.
+        /// The conversion constructor from text color representation.
+        /// hexcode representation : #RGB #RGBA #RRGGBB #RRGGBBAA
+        /// rgb representation : rgb(0-255,0-255,0-255) rgba(0-255,0-255,0-255,0.0-1.0)
         /// </summary>
-        /// <param name="hexColor">Hex color code</param>
+        /// <param name="textColor">color text representation as Hexcode, rgb() or rgba()</param>
         /// <exception cref="ArgumentNullException">This exception is thrown when hexColor is null.</exception>
         /// <since_tizen> 6 </since_tizen>
         /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public Color(string hexColor) : this(Interop.Vector4.NewVector4(), true)
+        public Color(string textColor) : this(Interop.Vector4.NewVector4(), true)
         {
             try
             {
-                if (null == hexColor)
+                if (null == textColor)
                 {
-                    throw new ArgumentNullException(nameof(hexColor));
+                    throw new ArgumentNullException(nameof(textColor));
                 }
-                hexColor = hexColor.Replace("#", "");
 
-                R = ((float)Convert.ToInt32(hexColor.Substring(0, 2), 16)) / 255.0f;
-                G = ((float)Convert.ToInt32(hexColor.Substring(2, 2), 16)) / 255.0f;
-                B = ((float)Convert.ToInt32(hexColor.Substring(4, 2), 16)) / 255.0f;
-                A = hexColor.Length > 6 ? ((float)Convert.ToInt32(hexColor.Substring(6, 2), 16)) / 255.0f : 1.0f;
+                textColor = textColor.ToLower();
```

Comment:
could you check warning message?

Reply:
Can you please let me know what is the warning ?

Reply:
this message->
src/Tizen.NUI/src/public/Common/Color.cs#L974
The behavior of 'string.ToLower()' could vary based on the current user's locale settings. Replace this call in 'Color.Color(string)' with a call to 'string.ToLower(CultureInfo)'.

Reply:
I see,
@huiyueun how do you compile TizenFX and see these errors ?
using
```
./build.sh full
```

or do you use Visual Studio or something else ?

Reply:
I build on Visual Studio, You can see this error.

Reply:
I see, then you build using Windows machine.
Are there any public docs that can help me building TizenFX on windows using Visual Studio?
or it is straight forward build